compiler/ballerina-parser/src/test/resources/declarations/annot-decl/annot_decl_assert_10.json

```diff
@@ -1,68 +1,68 @@
-                  "children": [
-                    {
-                      "kind": "SOURCE_KEYWORD"
-                    },
-                    {
-                      "kind": "TYPE_KEYWORD",
-                      "isMissing": true
-                    }
-                  ]
-                },
-                {
-                  "kind": "COMMA_TOKEN"
-                },
+            "kind": "LIST",
+            "children": []
+        },
+        {
+            "kind": "LIST",
+            "children": [
                 {
-                  "kind": "TYPE_KEYWORD",
-                  "isMissing": true
+                    "kind": "ANNOTATION_DECLARATION",
+                    "children": [
+                        {
+                            "kind": "METADATA",
+                            "children": [
+                                {
+                                    "kind": "LIST",
+                                    "children": []
+                                }
+                            ]
+                        },
+                        {
+                            "kind": "ANNOTATION_KEYWORD"
+                        },
+                        {
+                            "kind": "IDENTIFIER_TOKEN",
+                            "value": "Foo"
+                        },
+                        {
+                            "kind": "ON_KEYWORD"
+                        },
+                        {
+                            "kind": "LIST",
+                            "children": [
+                                {
+                                    "kind": "ANNOTATION_ATTACH_POINT",
+                                    "children": [
+                                        {
+                                            "kind": "SOURCE_KEYWORD"
+                                        },
+                                        {
+                                            "kind": "TYPE_KEYWORD",
+                                            "isMissing": true
+                                        }
+                                    ]
+                                },
+                                {
+                                    "kind": "COMMA_TOKEN"
+                                },
+                                {
+                                    "kind": "TYPE_KEYWORD",
+                                    "isMissing": true
+                                }
+                            ]
+                        },
+                        {
+                            "kind": "SEMICOLON_TOKEN"
+                        }
+                    ]
                 }
-              ]
-            },
-            {
-              "kind": "SEMICOLON_TOKEN"
-            }
-          ]
+            ]
+        },
+        {
+            "kind": "EOF_TOKEN"
         }
-      ]
-    },
-    {
-      "kind": "EOF_TOKEN"
-    }
-  ]
-}
\ No newline at end of file
+    ]
+}
```

Comment:
I know we earlier used to follow 4-spaces indentation for json formatting. But given our json generator tool can only support 2-spaces for indentation, shall we follow the same?
That way, it is easy to understand the diff went in to the file..

Reply:
Reformated.